src/server/scripts/Commands/cs_gobject.cpp

```diff
@@ -373,7 +363,7 @@ class gobject_commandscript : public CommandScript
             Unit* owner = ObjectAccessor::GetUnit(*handler->GetSession()->GetPlayer(), ownerGuid);
             if (!owner || !ownerGuid.IsPlayer())
             {
-                handler->PSendSysMessage(LANG_COMMAND_DELOBJREFERCREATURE, ownerGuid.ToString().c_str(), object->GetGUID().ToString().c_str());
+                handler->PSendSysMessage(LANG_COMMAND_DELOBJREFERCREATURE, ownerGuid.GetCounter(), guidLow);
```

Comment:
This is fixed by the other PR that tackles the types in general and fixes most if not all such differences in commands. https://github.com/TrinityCore/TrinityCore/pull/18863

However it uses tostring on the lowguid instead of the full guid. I guess in this case the correct way is as you say to use the tostring because its not a gameobject link.

But in general most of the code for commands gives out the lowguids instead of full guids since the commands use lowguid to refer to objects.
Hmm.